plugin/src/main/groovy/com/novoda/staticanalysis/internal/pmd/CollectPmdViolationsTask.groovy

```diff
@@ -1,7 +1,7 @@
 package com.novoda.staticanalysis.internal.pmd
 
 import com.novoda.staticanalysis.internal.CollectViolationsTask
-import com.novoda.staticanalysis.internal.Violations
+import com.novoda.staticanalysis.Violations
```

Comment:
Can you re-format all the files, I believe this import should come first.

Reply:
Such a eagle 👁